h/views/api_auth.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h.auth.services import TOKEN_TTL
+from h.exceptions import OAuthTokenError
+from h.util.view import json_view
+
+
+@json_view(route_name='token', request_method='POST')
```

Comment:
For now this will only be accessible for us, since 3rd parties will generate a JWT and pass that to the client, the client then takes the JWT and calls this endpoint.
I don't see why we wouldn't open this up to other developers, but that's still some months off.